commit 5c7fee0b2e636ad24594ab9e84a8e11ab2b0c6e0 (refs/changes/47/1047/2)
Author: Alexandra White <alexandra.white@joyent.com>
Date:   2016-12-05T09:49:57-05:00 (2 years, 10 months ago)
    
    Co-mention locality alongside Docker commands that mention affinity, hardwrapped at 80 characters

diff --git a/docs/api/commands/create.md b/docs/api/commands/create.md
index 60257dc..cc95d04 100644
--- a/docs/api/commands/create.md
+++ b/docs/api/commands/create.md
@@ -168,9 +168,11 @@ to size containers and assign public or private IP addresses to them.
 * `-v`, `--volume` and `--volumes-from` behave slightly differently in a Triton's container-native environment. See [volumes](../features/volumes.md).
 * `--volume-driver` and other `docker volume` commands are unimplemented at this time.
 
-Docker Swarm's affinity filters for controlling on which server a container
-is provisioned are supported. See the [placement feature
-documentation](../features/placement.md) for details.
+Docker Swarm's affinity filters  (also called "locality hints" in Triton, see
+the [cloudapi CreateMachine notes](https://apidocs.joyent.com/cloudapi/
+#CreateMachine)) for controlling on which server a container is provisioned are 
+supported. See the [placement feature documentation](../features/placement.md) 
+for details.
 
 Please contact Joyent support or file a ticket if you discover any additional divergence.
 
diff --git a/docs/api/commands/run.md b/docs/api/commands/run.md
index 819dd90..7fd9176 100644
--- a/docs/api/commands/run.md
+++ b/docs/api/commands/run.md
@@ -517,9 +517,11 @@ to size containers and assign public or private IP addresses to them.
 * `-v`, `--volume` and `--volumes-from` behave slightly differently in a Triton's container-native environment. See [volumes](../features/volumes.md).
 * `--volume-driver` and other `docker volume` commands are unimplemented at this time.
 
-Docker Swarm's affinity filters for controlling on which server a container
-is provisioned are supported. See the [placement feature
-documentation](../features/placement.md) for details.
+Docker Swarm's affinity filters  (also called "locality hints" in Triton, see
+the [cloudapi CreateMachine notes](https://apidocs.joyent.com/cloudapi/
+#CreateMachine)) for controlling on which server a container is provisioned are 
+supported. See the [placement feature documentation](../features/placement.md) 
+for details.
 
 
 ## Related
diff --git a/docs/api/features/placement.md b/docs/api/features/placement.md
index 70b1446..cfdf3f7 100644
--- a/docs/api/features/placement.md
+++ b/docs/api/features/placement.md
@@ -66,10 +66,11 @@ where node selection is described in terms of existing containers. For example:
     #                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     # Run a mysql instance that is NOT on the same node as container 'db0'.
 
-For the same reasons, node selection can matter on Triton. Triton's Docker
-implements the same affinity filters as documented for Docker Swarm, with the
-difference that you don't need to setup a Swarm cluster.
-
+For the same reasons, node selection can matter on Triton. Triton's Docker 
+implements the same affinity filters as documented for Docker Swarm, with the 
+difference that you don't need to setup a Swarm cluster. Affinity Filters are 
+also called "locality hints" in Triton (see the [cloudapi CreateMachine notes](
+https://apidocs.joyent.com/cloudapi/#CreateMachine)).
 
 ### Affinity filter syntax
 
